app/views/sephora_style_guide/home/_icons.slim

```diff
@@ -443,6 +443,148 @@ ruby:
       "size": "icon_30x20"
     }
   ]
+  countries = [
+    {
+      "name": "Hong Kong",
```

Comment:
Same thing here. If there is no specific order required, maybe alphabetical order?